inc/ocf_mngt.h

```diff
@@ -361,14 +361,9 @@ void ocf_mngt_cache_stop(ocf_cache_t cache,
  */
 struct ocf_mngt_cache_device_config {
```

Comment:
Maybe add a comment to cache attach API function (and load etc) stating that the volume passed in config structures is being "moved" from and is not usable after that.